### ISSUE TITLE
can't open normally, it happens nothing with blank window.

### ISSUE THREAD

**Issue by SevenPlusJun** [vletroye/SynoPackages#14]
Sun Mar 29 10:13:23 CST 2020 ------------------------------------------------------------------
Sun Mar 29 10:13:23 CST 2020 ROUTER.cgi has been called
Sun Mar 29 10:13:23 CST 2020 HANDLING request for /webman/3rdparty/MODS_PackageManager/mvpkg.php
Sun Mar 29 10:13:23 CST 2020 SCRIPT_FILENAME: /usr/syno/synoman/webman/3rdparty/MODS_PackageManager/mvpkg.php
Sun Mar 29 10:13:23 CST 2020 SCRIPT_PARAMETERS:
Sun Mar 29 10:13:23 CST 2020 REALPATH: /volume3/@appstore/MODS_PackageManager/ui/mvpkg.php
Sun Mar 29 10:13:23 CST 2020 EXECUTE: /usr/local/bin/php73-cgi -c /usr/local/etc/php73/cli/php.ini -d open_basedir=none /volume3/@appstore/MODS_PackageManager/ui/mvpkg.php  2>&1

the whole log is above. and it stock at the line " execute balabala".I really don't know why...

**Comment by SevenPlusJun** [comment]
Uncaught ReferenceError: $ is not defined
    at mvpkg.php:9

**Comment by SevenPlusJun** [comment]
#<script src="https://ajax.googleapis.com/ajax/libs/jquery/3.2.1/jquery.min.js"></script>
<script src="http://code.jquery.com/jquery-latest.js"></script>

I find the reason,  I can't visit the site google...

**Comment by SevenPlusJun** [comment]
however,in the end, it happens 
POST http://192.168.10.66:5002/webman/3rdparty/MODS_PackageManager/mvpkg.php 405 (Not Allowed)

**Comment by vletroye** [comment]
> Sun Mar 29 10:13:23 CST 2020 ------------------------------------------------------------------
> Sun Mar 29 10:13:23 CST 2020 ROUTER.cgi has been called
> Sun Mar 29 10:13:23 CST 2020 HANDLING request for /webman/3rdparty/MODS_PackageManager/mvpkg.php
> Sun Mar 29 10:13:23 CST 2020 SCRIPT_FILENAME: /usr/syno/synoman/webman/3rdparty/MODS_PackageManager/mvpkg.php
> Sun Mar 29 10:13:23 CST 2020 SCRIPT_PARAMETERS:
> Sun Mar 29 10:13:23 CST 2020 REALPATH: /volume3/@appstore/MODS_PackageManager/ui/mvpkg.php
> Sun Mar 29 10:13:23 CST 2020 EXECUTE: /usr/local/bin/php73-cgi -c /usr/local/etc/php73/cli/php.ini -d open_basedir=none /volume3/@appstore/MODS_PackageManager/ui/mvpkg.php 2>&1
> 
> the whole log is above. and it stock at the line " execute balabala".I really don't know why...

That's normal. Nginx is configured to handle any php page of the package with this router.cgi. 
The last message logged by router.cgi is that it executes php73-cgi with that page as param. Next, it ends.

**Comment by vletroye** [comment]
> #<script src="https://ajax.googleapis.com/ajax/libs/jquery/3.2.1/jquery.min.js"></script>
> 
> <script src="http://code.jquery.com/jquery-latest.js"></script>
> I find the reason, I can't visit the site google...

Indeed... I already read about other users, from China or Iran, experiencing issues with packages, like GateOne, because they couldn't access the JQuery of Google...

You can try with the one of Microsoft: <script src="https://ajax.aspnetcdn.com/ajax/jQuery/jquery-3.2.1.min.js"></script>
